cmd/get/clusters/printer.go

```diff
@@ -0,0 +1,71 @@
+package clusters
+
+import (
+	"io"
+
+	"github.com/giantswarm/microerror"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/cli-runtime/pkg/printers"
+
+	"github.com/giantswarm/kubectl-gs/cmd/get/clusters/provider"
+	"github.com/giantswarm/kubectl-gs/internal/key"
+	"github.com/giantswarm/kubectl-gs/pkg/output"
+)
+
+func (r *runner) printOutput(resource runtime.Object) error {
+	var (
+		err     error
+		printer printers.ResourcePrinter
+	)
+
+	switch {
+	case output.IsOutputDefault(r.flag.print.OutputFormat):
+		switch r.provider {
+		case key.ProviderAWS:
+			resource = provider.GetAWSTable(resource)
+		case key.ProviderAzure:
+			resource = provider.GetAzureTable(resource)
```

Comment:
We could have everything in a single function, but this allows full customization for each provider